Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -202,6 +202,76 @@ public void TestPutFileQuestionMarkWildcard()
             }
         }
         
+        [Test]
+        public void TestPutFileRelativePathWithoutDirectory()
+        {
+            // Prepare the data files to be copied
+            var prefix = Guid.NewGuid();
+            var file = $"{prefix}_1.csv";
+            PrepareFileData(file);
+            
+            // Set the PUT query variables
+            _inputFilePath = file;
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
```

Comment:
Do we need to call `open`? I would presume, that this is what `using` should perform.

Reply:
We do have explicitly `Open()` the connection as `using` doesn't do it implicitly.